lib/chitter.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+
+class Chitter_Peep
+
+  attr_reader :body, :time, :name, :handle
+
+  def initialize(body:, time:, name:, handle:)
+    @body = body
+    @time = time
+    @name = name
+    @handle = handle
+  end 
+
+  def self.view_all
+    if ENV['RACK_ENV'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else 
+      connection = PG.connect(dbname: 'chitter')
+    end 
+    result = connection.exec('SELECT * FROM peeps')
+    result.map do |peep|
+      Chitter_Peep.new(body: peep['body'], time: peep['time'], name: peep['name'], handle: peep['handle'])
+    end 
+  end 
+
+  def self.post_peep(body:, name:, handle:)
+    if ENV['RACK_ENV'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else 
+      connection = PG.connect(dbname: 'chitter')
+    end 
+    result = connection.exec("INSERT INTO peeps (body, time, name, handle) VALUES ('#{body}', '#{Time.now.round}', '#{name}', '#{handle}') RETURNING body, time, name, handle;")
```

Comment:
cool way of changing the time format

Reply:
This line is too long for me. Here's a link to creating multiline strings in ruby: https://commandercoriander.net/blog/2014/11/09/a-multiline-string-cheatsheet-for-ruby/

Something you could do for example is:
```rb
    query = <<-QUERY
INSERT INTO peeps (body, time, name, handle)
VALUES ('#{body}', '#{Time.now.round}', '#{name}', '#{handle}')
RETURNING body, time, name, handle;
   	QUERY
   	result = connection.exec(query)
```